subprojects/docs/src/samples/jvm-coverage-configurations/README.adoc

```diff
@@ -0,0 +1,32 @@
+NOTE: You can open this sample inside an IDE using the https://www.jetbrains.com/help/idea/gradle.html#gradle_import_project_start[IntelliJ native importer] or https://projects.eclipse.org/projects/tools.buildship[Eclipse Buildship].
+
+This sample shows how to use dependencies within a JVM multi-project build in Gradle. This is the same pattern used by all JVM languages (Java, Groovy, Scala).
```

Comment:
This looks like it's a copy from the multi-project build sample.  I think we need some "theory of operation" with the custom plugin here so people know what to expect.